recipes/raxml-ng/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "raxml-ng" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/amkozlov/{{ name|lower }}.git
+  git_rev: {{ version }}
+  sha256: 43b95ee1bddae0daee84644e9ee760a77f28bb16e2071e95cca79a30f39373e5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - gmp
+    - bison
+    - flex
+    - cmake
+    - pthread-stubs
```

Comment:
Is this actually needed? pthreads is there regardless.

Reply:
don't think that's necessary any more

Reply:
Removed